integration-tests/environments/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:launchMode="singleTask"
-        android:windowSoftInputMode="adjustResize">
+        android:windowSoftInputMode="adjustResize"
+        android:exported="true">
```

Comment:
Isn't this the fix? See https://developer.android.com/about/versions/12/behavior-changes-12#exported

Reply:
Ah yes sorry it should be false, good spot

Reply:
Except... the app will not start with it set to false 🙈

Reply:
@kneth I believe we need this to be `true`, because
>If the app component includes the [LAUNCHER](https://developer.android.com/reference/android/content/Intent#CATEGORY_LAUNCHER) category, set android:exported to true
and we do include that category: https://github.com/realm/realm-js/blob/e2919ec573e410e5fa630b79669856230939dfd2/integration-tests/environments/react-native/android/app/src/main/AndroidManifest.xml#L22

The app fails to start with `false` in any case

Reply:
My point is that this is more important than the `compileSdkVersion`.

Reply:
Oh I see, I had to upgrade the `compileSdkVersion` first to get to this error. Not sure why no one else was affected!